modules/nf-core/nanoplot/tests/main.nf.test

```diff
@@ -63,7 +63,36 @@ nextflow_process {
                     with(process.out.html.get(0)) {
                         assert get(1).collect { p -> file(p).getName() }.contains("NanoPlot-report.html")
                     }
-                } 
+                }
+            )
+        }
+
+    }
+
+    test("NanoPlot - stub") {
+
+        options "-stub"
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [ file(params.test_data['sarscov2']['nanopore']['test_sequencing_summary'], checkIfExists: true) ]
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match()
+                },
+                {
+                    with(process.out.html.get(0)) {
+                        assert get(1).collect { p -> file(p).getName() }.contains("NanoPlot-report.html")
+                    }
+                }
```

Comment:
```suggestion
                { assert snapshot(process.out).match() }
```

Checking for the html file seems redundant.

Reply:
Seems that the snapshot does not need to change for this